VideoLocker/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -55,7 +57,7 @@ public void setStart(String start) {
     }
 
     public String getCourse_image(Config config) {
-        return config.getApiHostURL() + course_image;
```

Comment:
Looks like returning nil is fine. We just pass the URL into ``Glide`` which does the right thing. But while I was messing with it, I found a couple more cases.